test/org.eclipse.elk.alg.topdown.test/src/org/eclipse/elk/alg/topdown/test/TopdownLayoutTest.java

```diff
@@ -52,7 +52,9 @@ public void testTwoLevelLayoutHorizontalScaling() {
         toplevel.setProperty(CoreOptions.PADDING, new ElkPadding());
         toplevel.setProperty(CoreOptions.SPACING_NODE_NODE, 0.0);
         // set size of node so that children will be scaled down
-        toplevel.setDimensions(20, 50);
+        graph.setProperty(CoreOptions.TOPDOWN_HIERARCHICAL_NODE_WIDTH, 20.0);
+        graph.setProperty(CoreOptions.TOPDOWN_HIERARCHICAL_NODE_ASPECT_RATIO, 0.4);
+//        toplevel.setDimensions(20, 50);
```

Comment:
Please elaborate why this is a comment (also see below).

Reply:
I guess it doesn't really add anything, I'll remove them.